avro-serializer/src/main/java/io/confluent/kafka/serializers/KafkaAvroDecoder.java

```diff
@@ -28,12 +28,17 @@ public KafkaAvroDecoder(SchemaRegistryClient schemaRegistry) {
     this.schemaRegistry = schemaRegistry;
   }
 
+    public KafkaAvroDecoder(SchemaRegistryClient schemaRegistry, boolean useSpecificDecoder) {
```

Comment:
Why have this extra constructor? The one above it is just for tests and any real use will need to override the default schema registry URL so it would have to use the one that takes the properties parameter, right?